argus/shell.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2016 Cloudbase Solutions Srl
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import argparse
+import os
+import subprocess
+import sys
+import tempfile
+
+from argus.backends.tempest import manager
+from argus.config import ci
+
+import requests
+import six
+from six.moves import urllib_parse as urlparse
+
+
+def _download_resource(url, location):
+    """Download a file from a remote url.
+
+    :param url: The URL that points to the resource
+    :param location: Where to save the resource
+    """
+    response = requests.get(url)
```

Comment:
Check the HTTP response code and also I think you should take care of any exceptions that may occur.